### ISSUE TITLE
How to use schedulers 

### ISSUE THREAD

**Issue by emited** [emited/tmd_framework#2]
It is not clear how to use schedulers in the Experiment, ie things that modify the state of the Experiment, in order to modify the learning rate, or temperatures across epochs, for example

**Comment by TomVeniat** [comment]
Is this issue specific to `EpochExperiment` ? 
If not, what do you mean by state of the experiment ?

**Comment by emited** [comment]
I was thinking about making the experiment have a state attribute, where all the parameters varying across training/evaluation where saved (including the metrics). But since Ignite's Engine does this already, i'm not sure if it is still usefull.
